fix(a11y): final fix for WCAG 1.1.1 image alt should describe image content accurately (#17)
scottrigby/basekamp-next: content/projects/stunt-double.md

attribution: Basekamp & friends
images:
  - src: insta3.jpg

>     alt: Artist painting large wall mural with colorful organic shapes in purple and green on black background, professional lighting and camera equipment visible

Reviewer: ```suggestion
    alt: Wall illustration of artist painting large colorful organic shapes in purple and green on black background, professional lighting and camera equipment visible in foreground
```

  - src: insta1.jpg
    alt: Gallery view showing "STUNT DOUBLE" text on wall with life-size photographic prints of people mounted on foam board in spotlight
  - src: open.jpg
    alt: Newspaper classified advertisement reading "WANTED! 200 extras for the production of STUNT DOUBLE" with details about showing up to Base Kamp dressed for art opening
  - src: close.jpg

>     alt: Repeated poster design showing four blue-tinted images of person crouching in white tank top with "STUNT DOUBLE" text and event information

Reviewer: ```suggestion
    alt: Repeated poster design showing four blue-tinted images of person crouching in white flightsuit with 'Stunt Double' text and event information
```

  - src: main.jpg
    alt: Two people standing in front of painted wall mural at ICA, snapshot showing green and orange geometric shapes
  - src: resp.jpg
    alt: Printed email with subject "Brilliant Work!" praising the conceptualization, with museum guest list visible at bottom
  - src: insta2.jpg
    alt: Gallery space view during installation showing person with lighting equipment, white walls being painted, and industrial ceiling
  - src: insta4.jpg
    alt: Artist painting wall with bold vertical stripes in orange, cream, and green colors in corner of industrial gallery space
  - src: insta5.jpg

>     alt: Wall mural in progress showing person painting large black organic shape on wall with red, yellow, and blue sections

Reviewer: ```suggestion
    alt: Wall mural showing two people painting large black organic shape on wall with red, yellow, and blue rectangular sections
```

  - src: sd1.jpg
    alt: Artists working on wall mural with vertical stripes in orange, cream, and green, showing process of painting Sol LeWitt-style composition
  - src: sd2.jpg
    alt: Gallery corner installation with colorful geometric wall murals and life-size photographic cutouts of people positioned against walls
  - src: sd3.jpg

>     alt: Installation view showing bold wall mural with red, yellow, black, and blue sections with two life-size photographic prints of people

Reviewer: ```suggestion
    alt: Wall mural with red, yellow, black, and blue sections with four life-size photographic prints of people appearing to paint the mural
```
